fp_val.py

```diff
@@ -1,20 +1,24 @@
-class FP:
-    val = ""
+from . import addon_updater_ops
+import requests
+import uuid
 
+class FP:
     def getVal(self, url):
+        try:
+            settings = addon_updater_ops.get_user_preferences()
```

Comment:
Did this work as you expected, saving the value in-between blender sessions?

Reply:
Yep, I closed it down and opened it back up and it was the same value